packages/nodejs/src/__tests__/config.test.ts

```diff
@@ -146,7 +146,9 @@ describe("Configuration", () => {
 
   describe("apiKey option", () => {
     it("sets the pushApiKey config option with the apiKey value", () => {
-      const warnMock = jest.spyOn(console, "warn").mockImplementation(() => {})
+      const warnMock = jest.spyOn(console, "warn").mockImplementation(() => {
+        return
```

Comment:
To avoid having to do this for these functions, it looks like we can also only allow these arrow functions. I would be fine with that. We use these types of empty functions a bunch. For "methods" explicitly calling `return` is good.

```js
    "@typescript-eslint/no-empty-function": [
      "error",
      { allow: ["arrowFunctions"] }
    ],
```

Reply:
That seems like a good compromise to me. 👍